api/src/stats/services/accounts.service.ts

```diff
@@ -0,0 +1,204 @@
+import { Injectable } from '@nestjs/common';
+import { ClickhouseService } from '../../clickhouse/clickhouse.service.js';
+import { StatsUtils } from '../utils/stats.utils.js';
+import { TimestampValue, AccountsStats } from '../interfaces/stats.interface.js';
+
+@Injectable()
+export class AccountsService {
+  constructor(
+    private readonly clickhouseService: ClickhouseService,
+    private readonly statsUtils: StatsUtils,
+  ) {}
+
+  async getAccountsOnChainOverTime(): Promise<TimestampValue[]> {
+    try {
+      const query = `
+        SELECT
+          version,
+          address
+        FROM olfyi.coin_balance
```

Comment:
SQL in this file seems inconsistent with the changes in files above (removing `olfyi.` prefix from table names).

Reply:
Good catch. Removed.